### ISSUE TITLE
GameGuru MAX - Request - Add Grid Plane when Terrain Hidden in Editor

### ISSUE THREAD

**Issue by MonkeyFrogStudio** [TheGameCreators/GameGuruRepo#5421]
Currently, if you turn off terrain in the editor, there's no way to see where the 'floor' is (0 on the Y axis), so it would be great to have a grid plane to show this. The grid plan should be transparent (only showing the grid lines) so that it's not obtrusive, making it easy for the end-user to place objects on, above, and below the grid plane.

**Comment by Necrym59** [comment]
You mean as in what i said in the original post??
"**When terrain option is turned off: -**
A semi-transparent and non collidable see through grid should be generated at 0,0,0 outwards to the size of the playable map area. Only seen in the editor, it is used for a guide for positioning of objects all new objects i should expect would be dropped at 0,0,0 (centre) in respect to the screens viewpoint at the time. Positioning will then be a free movement within the 3d world scope. Snapping can be handled by the standard grid handling if need be.

**Comment by MonkeyFrogStudio** [comment]
Yes, @Necrym59. Remember, Lee asked me to create this as a separate request.

**Comment by Necrym59** [comment]
And im just adding the comment so its not lost as well an mey be pertinent

**Comment by OldFlak** [comment]
The camera also needs to be able to go under the terrain in the editor even when not wanting to hide terrain at all.
- makes it tedious when placing items in basements etc that are in a hole in the ground.
